src/storage/README.md

```diff
@@ -47,9 +51,30 @@ version. These features will be rolled out in future updates._
 ## Reading an encrypted file
 
 ```JavaScript
- blockstack.getFile("/message.txt", true)
+ let options = { 
+   decrypt: true 
+ }
+
+ blockstack.getFile("/message.txt", options)
  .then((fileContents) => {
     // get & decrypt the contents of the file /message.txt
     assert(fileContents === "Secret hello!")
  });
 ```
+
+## Reading another user's unencrypted file
+In order for files to be publicly readable, the app must request 
+the `publish_data` scope during authentication.
```

Comment:
I like that `publish_data` is similar to what facebook calls their write permission: https://developers.facebook.com/docs/facebook-login/permissions#reference-publish_actions